packages/attribute-slicer/src/selection/JQuerySelectionManager.ts

```diff
@@ -188,6 +169,10 @@ export default class JQuerySelectionManager<T extends ISelectableItem<any>> exte
                 })
                 .on(`click${EVENTS_NS}`, function (e) {
```

Comment:
If you use an arrow function here it will bind `this`, so you won't need to use `that`